### PR TITLE
doc: update about astar page

### PR DESCRIPTION
**Pull request by SUGURUSEO**
Added a small description and a developer incentive section in about astar page as this was mentioned from @SotaWatanabe

### CODE REVIEW

File: docs/about/astar.md [hunk 1]
```diff
@@ -25,3 +26,6 @@ Scalability on the Astar Network is addressing the followings topics:
 Many different blockchains exist but few have interoperability as a feature. To move "value" from Bitcoin to Ethereum, we need a centralized exchange. Polkadot is a sharded protocol that enables blockchain networks to operate together seamlessly.
 
 [Astar Network] is compatible with Polkadot. By connecting Astar Network to Polkadot, we will have true interoperability.
```
Comment (andabakus) [inline docs/about/astar.md:28]: This sounds a little bit outdated - "By connecting Astar to Polkadot we will have true interoperability" - maybe rephrase to present tense (I assume this text is from pre-winning parachain slot time)

Comment (SUGURUSEO) [reply docs/about/astar.md]: Thank you for the feedback. I agree with your opinion and changed it to present tense.

**Comment by SUGURUSEO** [comment]
@andabakus I appreciate if you can approve this PR too. I made a necessary change.

**Review by andabakus — APPROVED** [review]
LGTM
